apps/emqx_mt/src/emqx_mt_api.erl

```diff
@@ -193,9 +323,117 @@ client_count(get, #{bindings := #{ns := Ns}}) ->
         {error, not_found} -> ?NOT_FOUND("Namespace not found")
     end.
 
+tenant_limiter(get, #{bindings := #{ns := Ns}}) ->
+    %% TODO: check NS exists and was explicitly created.
```

Comment:
This and similar occurrences are probably best handled in follow ups, when we implement more of the config?